projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html

```diff
@@ -18,7 +18,7 @@
 
 <ng-template #pageHeader>
     <div>
-        <img src="../../../assets/img/netgrif_full_white.svg" alt="logo" class="logoimg">
+        <!-- <img src="TODO LINK" alt="logo" class="logoimg"> -->
```

Comment:
This is only an example header image, so if this line is not used, the commented line can be removed.

Reply:
Depends on https://github.com/netgrif/components/pull/94